### DISCUSSION TITLE
💡

### DISCUSSION THREAD

**Discussion by Nikhiluuuuuuuuu** [MissingCore/Music#85] [Ideas]
To enhance your music experience, consider adding lyrics to your favorite songs and creating dedicated pages for each artist. Platforms like Spotify, YouTube Music, and Apple Music offer these features. Search for the song you want, then look for the option to add lyrics. Input the lyrics and save changes. Use playlists to organize songs by artist, adding all their songs to a playlist and naming it accordingly. For songs that require additional details, create a note or tag. Websites and apps can help you find lyrics if you're unsure. Add descriptions to playlists for more context about the artist.

**Comment by cyanChill** [comment]
There's no plans for adding lyric support of any kind - you can [read this](https://github.com/MissingCore/Music/discussions/123#discussioncomment-11683206) for some of the reasons why. We already support playlists and have dedicated pages for artists listing out their albums and tracks.
